src/main/java/com/epam/poc/wearecommunity/core/hook/Hook.java

```diff
@@ -21,13 +21,15 @@ public Hook() {
     }
 
     @Before
-    public void setUp() {
+    public void setUp(Scenario scenario) {
         String browserProperty = System.getProperty("Browser");
         String browserName = (browserProperty != null) ? browserProperty : new PropertyReader(GlobalConstants.CONFIG_FILE_KEY).getValue(GlobalConstants.BROWSER_NAME).toUpperCase();
 
         String pageUrl = new PropertyReader(GlobalConstants.CONFIG_FILE_KEY).getValue(GlobalConstants.PAGE_URL);
 
         logger.info("Running on page URL: " + pageUrl + " with browser name: " + browserName);
+        scenario.attach("Title scenario that included browser name", "text/plain",
+                browserName.toUpperCase() + "-" + scenario.getName());
```

Comment:
Chỗ này có cần "scenario.getName()" không em? Vì anh thấy trên report nó đang hiển thị duplicate tên vs line bên trên của attach

Reply:
Em search "@Smoke" rồi replace toàn bộ bằng @Regression với nhé

Reply:
Em đã có push code update dựa theo review của anh rồi, anh review lại giúp em commit cuối cùng nha. Thank anh.